src/frontend/components/SøknadsSteg/Kvittering/innholdTyper.ts

```diff
@@ -0,0 +1,5 @@
+import { LocaleRecordBlock } from '../../../typer/sanity/sanity';
+
+export type IKvitteringTekstinnhold = {
```

Comment:
Jeg er ikke så glad i prefixers slik som "I" i språk som java og typescript da typen allerede er definert her som "type", men ser det er brukt en del allerede så trenger ikke gjøre noe med det nå. 

Se f.eks. https://stackoverflow.com/questions/111933/why-shouldnt-i-use-hungarian-notation 

"Hungarian notation only makes sense in languages without user-defined types. In a modern functional or OO-language, you would encode information about the "kind" of value into the datatype or class rather than into the variable name."

Reply:
Jeg er enig med deg, jeg finner "I" prefix på interfaces unødvendig, men som du sa blir det brukt de fleste andre steder. Jeg har derfor valgt å bruke det slik at det blir konsistent